kube-model.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions/v1beta1
```

Comment:
Do we want to have this duplicated here? It's already available at [image-caption-generator](https://github.com/IBM/MAX-Image-Caption-Generator/blob/master/image-caption-generator.yaml) repository